test-apps/imodel-from-reality-model/src/RealityModelContextIModelCreator.ts

```diff
@@ -1,3 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
 /*---------------------------------------------------------------------------------------------

```

Comment:
Not sure what happened with copyright header here, maybe Caleb's script went bonkers.

Reply:
This is fixed on master now, but you'll need to run the fix locally to revert changes in already committed files.

Please run the following locally to revert these changes:

```
git co master
git pull
git co <your branch>
git merge master
rush fix-copyrights
```

Reply:
I'll do it. FYI @RBBentley by `co` he means `checkout`.

Reply:
@wgoehrig done.